capsules/segmenter_person_openvino/capsule.py

```diff
@@ -0,0 +1,40 @@
+from vcap import (
+    BaseCapsule,
+    NodeDescription,
+    DeviceMapper,
+    common_detector_options,
+    BoolOption
+)
+from vcap_utils import BackendRpcProcess
+from .backend import Backend
+
+
+class Capsule(BaseCapsule):
+    name = "segmenter_person_openvino"
+    description = "✨ OpenVINO person segmenter."
+    version = 1
+    device_mapper = DeviceMapper.map_to_openvino_devices()
+    input_type = NodeDescription(size=NodeDescription.Size.NONE)
+    output_type = NodeDescription(
+        size=NodeDescription.Size.ALL,
+        detections=["person"])
+    backend_loader = lambda capsule_files, device: BackendRpcProcess(
+        Backend,
+        model_xml=capsule_files["instance-segmentation-security-0050.xml"],
+        weights_bin=capsule_files["instance-segmentation-security-0050.bin"],
+        device_name=device
+    )
+    options = {
+        **common_detector_options,
+        "keep_aspect_ratio": BoolOption(
+            default=False,
+            description="If True, it will keep the aspect ratio of "
```

Comment:
Nit: I'm not sure of the technical term for this, but don't use "it will". The "it" can lead to confusion.

ex:
"If True, it will keep the aspect ratio" -> "If True, the aspect ratio will be kept"